homeassistant/components/plex/media_player.py

```diff
@@ -58,13 +62,17 @@
 
 def setup_platform(hass, config, add_entities_callback, discovery_info=None):
     """Set up the Plex platform."""
-    if PLEX_DATA not in hass.data:
-        hass.data[PLEX_DATA] = {}
+    if PLEX_DOMAIN not in hass.data:
+        hass.data[PLEX_DOMAIN] = False
+
+    # Check if already configured
+    if hass.data[PLEX_DOMAIN]:
+        return
 
     # get config from plex.conf
     file_config = load_json(hass.config.path(PLEX_CONFIG_FILE))
 
-    if file_config:
+    if file_config and config:
```

Comment:
Why do we depend on `config` here?

Reply:
Not needed with the previous check added.